Misc/NEWS.d/next/Core and Builtins/2024-05-03-18-01-26.gh-issue-95382.73FSEv.rst

```diff
@@ -0,0 +1 @@
+Improve performance of :func:`json.dumps` and :func:`json.dump` when using the argument `indent`.
```

Comment:
```suggestion
Improve performance of :func:`json.dumps` and :func:`json.dump` when using the argument *indent*.
```
Also, would be nice to give a rough number (5%? 20%?).

Reply:
Wouldn't it be better to say `indent` is supported in the C implementation? We also didn't mention performance numbers in any of my previous pull requests.

```suggestion
Support *indent* in the C implementation of :func:`json.dumps` and :func:`json.dump`.
```

Reply:
NEWS entries should be focused on the user-visible effect, not implementation details.

Reply:
> Also, would be nice to give a rough number (5%? 20%?).

Number are always tricky. For `dumps` the performance improvement is a factor 2.62 (on the benchmark tested). I would say the performance is then 160% faster. But for `dump` the performance improvement will be less, and dependent on whether one is writing to a file or stdout.

Reply:
It is data depending, but for several simple examples I get ~3 time speed up. It is close to the speed of serialization without indentation.

Reply:
Those are great numbers! That seems significant enough that it's worth calling out in the NEWS entry, and maybe also in the What's New for 3.13.

Reply:
I updated the numbers in the news entry. The whats new can wait I think untill after the 3.13 beta release.